tests/Unit_Validator.py

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+import astropy.units
+
+class Unit_Validator():
```

Comment:
Please rename to `Unit_Validator` (I think pep8 checkers will complain otherwise) and remove the `()` if it's really not a subclass.

Reply:
Completely redid this implementation. The check_unit() function is now in test_utils.py.